mods/cnc/rules/structures.yaml

```diff
@@ -78,7 +79,8 @@ FACT.GDI:
 		ForceFaction: gdi
 	Tooltip:
 		Name: Construction Yard (GDI)
-
+	AcceptsDeliveredCash:
```

Comment:
This and one on FACT.NOD is not required. They already Inherit that from FACT.

Actually you can just put it on ^Building under defaults.yaml and remove it on ^Defence and others that don't need it as done in RA mod. No need to specify it on each building.